build.sbt

```diff
@@ -109,6 +111,8 @@ lazy val chiselFrontend = (project in file("chiselFrontend")).
   ).
   dependsOn(coreMacros)
 
+// Hack to avoid circular dependency on chisel.
+val root = Project("chisel", file("."))
 
 lazy val chisel = (project in file(".")).
```

Comment:
I have no idea if projects actually do redefine it, I was just speculating about how this works! Does `unmanagedBase in root` mean this `val root` or does it mean like the actual "root" project?